crates/pallet-domains/src/lib.rs

```diff
@@ -1221,12 +1223,18 @@ mod pallet {
             domain_id: DomainId,
             amount: BalanceOf<T>,
             config: OperatorConfig<BalanceOf<T>>,
+            signing_key_proof_of_ownership: OperatorSignature,
```

Comment:
This can put the burden on the user when registering operator, e.g. how do I get this `signing_key_proof_of_ownership` when registering operator on polkadot.js, perhaps provide a helpful tool under the `subspace-node domain` command?

Reply:
This was initial concern as well but we decided to update our staking UI to handle the signature generation part rather providing a cli. We leaned towards that in the discussion here - https://github.com/subspace/subspace/issues/2554